sample/src/main/java/com/chuckerteam/chucker/sample/HttpBinClient.kt

```diff
@@ -145,4 +149,32 @@ class HttpBinClient(
             }
         )
     }
+
+    private fun getProtoResponse() {
+        val pokemon = Pokemon("Pikachu", level = 99)
+        val body = pokemon.encodeByteString().toRequestBody("application/protobuf".toMediaType())
+        val request = Request.Builder()
+            .url("https://postman-echo.com/post")
```

Comment:
This class is getting bloated :( It's called `HttpBinClient` but we do a lot of other stuff inside:

```
        downloadSampleImage(colorHex = "fff")
        downloadSampleImage(colorHex = "000")
        getResponsePartially()
        getProtoResponse()
```

Can you please move this code into `PostmanEchoClient` and then we'll take care of refactoring the status quo?

Reply:
I didn't want to move focus from decoding in this PR. Like mentioned in "next steps" section I'd prefer to restructure it in a separate PR if you don't mind.

Reply:
Yup let's follow up on this